requirements.txt

```diff
@@ -5,10 +5,10 @@ git+https://github.com/DIRACGrid/pyGSI.git#egg=GSI
 # From pypi
 autopep8==1.3.3
 certifi
-CMRESHandler>=1.0.0b4
 coverage
 codecov
-elasticsearch-dsl>=2.0.0
+elasticsearch-dsl~=6.3.1
+CMRESHandler>=1.0.0b4
```

Comment:
@atsareg @andresailer I think these changes fix the issues that Travis show on other PRs, like those seen in https://travis-ci.org/DIRACGrid/DIRAC/jobs/520371438